samples/ObjCDemoApp-Framwork/Podfile

```diff
@@ -0,0 +1,7 @@
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '9.0'
+
+target 'ObjCDemoApp' do
+  pod 'GoogleMaps'
+# pod 'Google-Maps-iOS-Utils', :path => '../..'
```

Comment:
Why can this be commented out? It's best to demonstrate usage using Cocoapods with `use_frameworks!` enabled.